### PR TITLE
Rebuild python3801 aarch parallel2

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,29 +1,30 @@
...
+    - scipy-1.3.2-bad-tests.patch  # [osx and py == 38]
+    - gh11046.patch                # [ppc64le]
+{% endif %}
 
 
 build:
-  number: 2
+  number: 0
   skip: true  # [win or py2k]
+  skip: true  # [not aarch]
```
Comment (hmaarrfk) [inline recipe/meta.yaml]:
```suggestion
  skip: true  # [not aarch64]
```

File: conda-forge.yml [hunk 2]
```diff
@@ -3,6 +3,6 @@ appveyor:
 compiler_stack: comp7
 max_py_ver: '37'
 max_r_ver: '35'
-provider: {linux_aarch64: default, linux_ppc64le: default}
+provider: {linux_aarch64: azure, linux_ppc64le: default}
```
Comment (hmaarrfk) [inline conda-forge.yml]:
```suggestion
provider: {linux_aarch64: default, linux_ppc64le: default}
```

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

**Comment by beckermr** [comment]
This can be closed, right @hmaarrfk ?

**Pull request closed without merging**